src/modules/xcm/index.ts

```diff
@@ -217,6 +217,24 @@ export const xcmChainObj: XcmChainObj = {
     subscan: 'https://bifrost-kusama.subscan.io/',
     isAstarNativeToken: true,
   },
+  [Chain.EQUILIBRIUM]: {
+    name: Chain.EQUILIBRIUM,
+    relayChain: Chain.POLKADOT,
+    img: 'https://contentv2.equilibrium.io/uploads/eq_f2099c1422.svg',
```

Comment:
Please add the logos to `/src/assets/img/chain` and change this line like ` img: require('/src/assets/img/chain/equilibrium.png')`